core/src/plugins/filed/python/vmware/BareosFdPluginVMware.py

```diff
@@ -288,6 +316,15 @@ def start_backup_job(self):
         if not self.vadp.connect_vmware():
             return bareosfd.bRC_Error
 
+        if "uuid" in self.options:
```

Comment:
Does it make sense to cancel the backup or issue a warninig as uuid is now useless?

Reply:
The uuid option still works as in the old version, but restore to other dc, folder, datastore is not possible and trying it will be rejected by the plugin now. Added that in docs.